python/revng/internal/daemon/__init__.py

```diff
@@ -189,6 +202,7 @@ def shutdown():
         routes=routes,
         on_startup=[startup],
         on_shutdown=[shutdown],
+        exception_handlers={ClientDisconnect: client_disconnect_handler},  # type: ignore
```

Comment:
Have you been able to test this?

Reply:
I've tried but I haven't been able to yank the connection hard enough to make it trigger "naturally"
Manually triggering the exception does however trigger it, so I'm confident it works